src/executor/executorManager.ts

```diff
@@ -375,11 +443,40 @@ export class ExecutorManager {
     async refreshUserOperationStatuses(): Promise<void> {
         const ops = this.mempool.dumpSubmittedOps()
 
-        const txs = getTransactionsFromUserOperationEntries(ops)
+        const uniqueEntryPoints = new Set<Address>(
+            ops.map((op) => op.userOperation.entryPoint)
+        )
+
+        const opEntryPointMap = new Map<Address, SubmittedUserOperation[]>()
+
+        for (const op of ops) {
+            if (!opEntryPointMap.has(op.userOperation.entryPoint)) {
+                opEntryPointMap.set(op.userOperation.entryPoint, [])
+            }
+            opEntryPointMap.get(op.userOperation.entryPoint)?.push(op)
+        }
```

Comment:
we already know all possible EntryPoints from the config. can we just use it here? without cycling through each userop

and even with current logic we could make it with one iteration

Reply:
oh yeah good point, forgot about it!

Reply:
also we do need to loop over all the ops, because we will be bundling them below. so we need to first make the map of 
```typescript
{ 
   [entryPoint]: ops[]
}
```

and then use this map to send the complete bundle of ops